backend/pkg/count/service/count_service.go

```diff
@@ -198,26 +190,32 @@ func (cs *CountService) getCoOccurringCluster(
 	return coOccurringClusters, nil
 }
 
-func (cs *CountService) getOccurrencesOfClusterId(
+func (cs *CountService) increaseOccurrencesForMisses(
 	ctx context.Context,
 	clusterId string,
-	fromTime time.Time,
-	toTime time.Time,
-) (int64, error) {
-	// *2 because we want to search for a decent sized window around the timestamp
-	queryCtx, cancel := context.WithTimeout(ctx, csTimeOut)
-	defer cancel()
-	query := countOccurrencesQueryBuilder(clusterId, fromTime, toTime)
-	queryBody, err := json.Marshal(query)
+	coOccurringClusters map[string][]model.Cluster,
+) error {
+	listOfCoOccurringClusters := make([]string, len(coOccurringClusters))
+	i := 0
+	for key, _ := range coOccurringClusters {
+		listOfCoOccurringClusters[i] = key
```

Comment:
This is dumb but Golang forces me to do this. Normally I would just say coOccurringClusters.keys()